ckanext/dia/harvester/csw.py

```diff
@@ -206,6 +210,54 @@ def get_package_dict(self, context, data_dict):
 
         package_dict['groups'] =  dict((group['id'], group) for group in groups).values()
 
+        # CSW records can have a non wgs-84 projection, we will need to convert the geojson to wgs-84
+        for extra in package_dict['extras']:
+            if extra['key'] == 'spatial-reference-system':
+                spatial_srid = extra['value']
+            if extra['key'] == 'spatial':
+                try:
+                    spatial_geojson = json.loads(extra['value'])
+                except ValueError:
+                    log.warn('Failed to parse json for spatial field of package {}'.format(package_dict))
+
+        if not not spatial_srid and spatial_geojson is not None:
```

Comment:
I haven't seen `not not` in python before, so that's interesting.
I'm assuming this doesn't work if it's
`if spatial_srid` instead?
`if bool(spatial_srid)` is also an option.

Reply:
This could probably be a bit clearer. This weird conditional is a result of the following.
- sometimes spatial_srid is the empty string `''`
- the empty_string == False.


I think that `if bool(spatial_srid)` is a refactor that makes sense on this one

Reply:
I can't seem to work out how:
`if spatial_srid` is  functionally different than `if bool(spatial_srid)` or `if not not spatial_srid`
They're both `False` when `spatial_srid` is `''` or `None`
They're both true when `spatial_srid` has any characters?

I think i'd generally prefer `if spatial_srid`
What am I missing?